deps.versions.toml

```diff
@@ -1,25 +1,25 @@
 [versions]
 
 decompose = "2.0.1-compose-experimental"
-kotlin = "1.8.20"
+kotlin = "1.8.21"
 essenty = "1.1.0"
 parcelizeDarwin = "0.1.4"
 reaktive = "1.2.3"
 junit = "4.13.2"
-jetbrainsCompose = "1.4.0"
+jetbrainsCompose = "1.5.0-beta01"
 jetbrainsKotlinWrappers = "1.0.0-pre.478"
-jetbrainsKotlinxCoroutines = "1.6.4"
+jetbrainsKotlinxCoroutines = "1.7.1"
 jetbrainsBinaryCompatibilityValidator = "0.12.1"
-jetpackCompose = "1.4.0"
-jetpackComposeCompiler = "1.4.5"
-androidGradle = "7.2.2"
-androidMaterial = "1.6.1"
+jetpackCompose = "1.5.0-beta01"
+jetpackComposeCompiler = "1.5.1"
+androidGradle = "7.3.1"
```

Comment:
I'm trying to keep this version in sync with https://github.com/arkivanov/gradle-setup-plugin/blob/master/build.gradle.kts#L23. If possible, let's keep this unchanged, I will update AGP on my own later.

Reply:
According to previous comment, the versions of `jetpackCompose` and `jetpackComposeCompiler` should be matched to the ones used in Compose MP (which is not the case here). I can update the versions, but as you mentioned, additional updates for keeping projects in sync would be necessary if updating the AGP.

I couldn't find any compatibility information about AGP, but it seems to work fine with 7.2.2. So I can revert this update too. :)